lib/quadrigacx/coin.rb

```diff
@@ -0,0 +1,19 @@
+module QuadrigaCX
+  module Coin
+    BITCOIN = 'bitcoin'
+    LITECOIN = 'litecoin'
+    ETHER = 'ether'
+
+    def self.valid? type
+      ALL_COINS.include?(type)
+    end
+
+    private
+
+    ALL_COINS = [
```

Comment:
Would it make sense to use a list of symbols here? Like so:

```
COINS = %i[
  bitcoin 
  litecoin 
  ether
]
```

Reply:
I was originally thinking that but figured if there was some variation in the coin string and the name the current approach would account for that better.

As it stands now there isn't really a case like that. Recent changes have `Bitcoin Cash`, so even that could be a symbol (:bitcoin_cash).

I'm indifferent to either approach.

Reply:
What you have now works for me.